tests/test_attribute_selector_block.py

```diff
@@ -0,0 +1,51 @@
+from nio.block.terminals import DEFAULT_TERMINAL
+from nio.signal.base import Signal
+from nio.testing.block_test_case import NIOBlockTestCase
+from ..attribute_selector_block import AttributeSelector
+
+
+class TestExample(NIOBlockTestCase):
+
+    def test_blacklist_signals(self):
```

Comment:
I think this test would fail:
```python
def test_hidden_attributes(self):
    blk = AttributeSelector()
    self.configure_block(blk, {'specify_behavior': 'WHITELIST',
                               'specify_attributes': ['_hello']})
    blk.start()
    blk.process_signals([Signal({'_hello': 'n.io', 'goodbye': 'n.io'})])
    blk.stop()
    self.assert_num_signals_notified(1)

    self.assertDictEqual(
        self.last_notified[DEFAULT_TERMINAL][0].to_dict(),
        {'_hello': 'n.io'})
```